ArduinoCode/Pedal_Potentiometers/Pedal_Potentiometers.ino

```diff
@@ -73,10 +73,17 @@ void readPotentiometers() {
   if (abs(accelPin1Val - accelPin2Val) <= (1024 * 0.1)) {
```

Comment:
Before this if statement, there should be an if statement that checks if inverterOn is false. If it is, return out of the function as we don't want to send any messages to the MC when the inverter is supposed to be off.